### PR TITLE
Translating docs/recipes/styling-css.md

### PR DESCRIPTION
**Pull request by k0mpreni**

### CODE REVIEW

File: docs/docs/recipes/styling-css.md [hunk 1]
```diff
@@ -25,46 +22,45 @@ p {
...
 
 ```javascript:title=gatsby-browser.js
 import "./src/styles/global.css"
 ```
 
-> **Note:** You can also make use of `require('./src/styles/global.css')` to import the global CSS file in your `gatsby-config.js` file.
+> **Note:** Vous pouvez aussi utiliser `require('./src/styles/global.css')` pour importer la feuille de style globale dans votre fichier gatsby-config.js`
 
-3. Run `gatsby-develop` to observe the global styling being applied across your site.
+3. Lancer `gatsby-develop` pour observer le style global appliqué à votre site.
```
Comment (dantereve) [inline docs/docs/recipes/styling-css.md]:
```suggestion
3. Lancez `gatsby-develop` pour observer le style global appliqué à votre site.
```

**Review by dantereve — CHANGES_REQUESTED** [review]
Woah, that's a pretty big one for your first contribution to Gatsby! 🚀Thanks a lot!  I hope you had fun translating this page. 

I've only spotted one tiny thing to change before I can merge this. You can commit directly from this PR. 😉

File: docs/docs/recipes/styling-css.md (hunk 1 above)
Comment (k0mpreni) [reply docs/docs/recipes/styling-css.md]: Haha I was working on adding style on Gatsby at this moment so nice to read and learn while translating it!

**Review by mindsers — APPROVED** [review]
🚀

**Review by dantereve — APPROVED** [review]

**Pull request merged**